src/shared/renderUtils.jsx

```diff
@@ -1,19 +1,58 @@
 import React from "react";
 
-function renderBody(body){
-	const paragraphs = body.split("\n"); // splits the string at every line-break
-	return paragraphs.map(renderParagraph);
+function renderBody(data, options = {}){ 
+	const paragraphs = data.split("\n"); // splits the string at every line-break
+	const customisedRenderParagraph = renderParagraph.bind(this, options);
+
+	return paragraphs.map(customisedRenderParagraph);
 }// renderBody
 
-function renderParagraph(text, index){
-	const key = `about__body__paragraph_${index}`;
+
+function renderParagraph(options, text, index){
+	const { scope, className } = options;
```

Comment:
```suggestion
	const { 
		scope, 
		className = ""
	} = options;
```
Good to default `className` to `""` otherwise we end up with elements with loads of `<div class="undefined">` and stuff like that.